### PR TITLE
Ember page - add new styles

### PR DESCRIPTION
**Pull request by hannakim91**
Incorporate new styles from [Figma](https://www.figma.com/file/faY8Y71bcubIyke4sbiNNE/Ship-Shape-Website?node-id=957%3A955)

**To Do:**
- [x] Which case study for paragraph starting with  “For these reasons, we often recommend…" - [See here](https://docs.google.com/document/d/1KtgEcMD2F4h3cBhjmVRQYTzTWqdB5LnPpWWAoGJWx5o/edit#).

### CODE REVIEW

File: pages/services/ember-consulting.vue [hunk 1]
```diff
@@ -1,86 +1,212 @@
...
-          class="h-auto m-auto max-w-2xl w-full"
-          format="webp"
-          height="858"
-          src="/img/hero-images/ember-consulting.png"
-          width="1089"
-        />
-      </template>
-    </HeroBlock>
+    <div class="waves white-waves sm:pb-20 lg:pb-8">
+      <HeroBlock class="ember-consulting">
```
Comment (rwwagner90) [inline pages/services/ember-consulting.vue:4]: @hannakim91 I would recommend looking closely at the next.js page in the main branch for the format for things. For example, this `HeroBlock` should not be wrapped in a div, and it takes `hero-classes` to apply classes to it.

Comment (hannakim91) [reply pages/services/ember-consulting.vue:4]: Thanks, didn't realize the difference was `hero-classes` to get styles to apply to that

**Comment by hannakim91** [comment]
@rwwagner90 which case study do you want for the "enterprise app development" keywords?

File: pages/services/ember-consulting.vue [hunk 2]
```diff
@@ -1,86 +1,216 @@
...
-    <RecentBlogPosts :posts="posts" />
+    <WaveMoreLinks class="md:px-20" />
   </div>
 </template>
 
 <script>
 import { generateMeta } from '~/utils/meta';
 
+const testimonial = {
+  name: 'Namey Nameson',
```
Comment (rwwagner90) [inline pages/services/ember-consulting.vue:197]: @hannakim91 for this testimonial, can you please update it to be the one from Roger from Expel we use on the homepage?

**Review by rwwagner90 — COMMENTED** [review]
@hannakim91 This is almost good to go! I think we're just missing the "ember.js in action" section and the testimonial needs to be updated to be the one from Roger from Expel please.

For the ember.js in action one we need to create a new component to show those 3 case studies like that. Ideally, the component would support 1, 2, or 3 case studies and change its design to fit what is in the style guide under "case study layout" https://www.figma.com/file/faY8Y71bcubIyke4sbiNNE/Ship-Shape-Website?node-id=0%3A1

It could also be separate components for the different numbers though. Whatever you think works best!

**Pull request merged**